main.nf

```diff
@@ -9,6 +9,9 @@ workflow {
         RUN()
     } else if (params.mode == "run_validation") {
         RUN_VALIDATION()
+    } else if (params.mode == "test") {
+        INDEX()
+        RUN()
```

Comment:
Add RUN_VALIDATION (and turn on BLAST for the run workflow)

Reply:
Remove this, but did turn on blast, I still need to test run validation.